gcloud/_helpers.py

```diff
@@ -394,6 +394,27 @@ def _to_bytes(value, encoding='ascii'):
         raise TypeError('%r could not be converted to bytes' % (value,))
 
 
+def _from_bytes(value):
```

Comment:
This might need a clearer name... it's converting bytes to unicode? `_from_bytes` doesn't really convey that ...